packages/truffle-events/defaultSubscribers/compile.js

```diff
@@ -0,0 +1,61 @@
+const OS = require("os");
+
+module.exports = {
+  initialization: function() {
```

Comment:
Actually I did it to avoid context issues. The logger wasn't binding properly. Probably a silly problem but I didn't want to spend too much time on it at the time. I would rather use them though as it makes the code feel a little less dense.

Reply:
@davidmurdoch Do you know offhand what a better solution for this would be? I guess what really needs to happen is that `this` needs to bind to the class which doesn't happen with arrow functions. I guess we could just bind them manually in the class, then it wouldn't matter and these descriptions could be prettier.

Reply:
Hmm. If they are arrow functions you can't rebind them later, AFAIK. Alternatively, we could pass a `context` to the `initialization` function and the `handler` functions. Let's chat about this.